contracts/src/periphery/EigenDACertVerifierV3.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifier} from "src/interfaces/IEigenDACertVerifier.sol";
+import {IEigenDAThresholdRegistry} from "src/interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDABatchMetadataStorage} from "src/interfaces/IEigenDABatchMetadataStorage.sol";
+import {IEigenDASignatureVerifier} from "src/interfaces/IEigenDASignatureVerifier.sol";
+import {OperatorStateRetriever} from "lib/eigenlayer-middleware/src/OperatorStateRetriever.sol";
+import {IRegistryCoordinator} from "lib/eigenlayer-middleware/src/interfaces/IRegistryCoordinator.sol";
+import {EigenDACertVerificationV3Lib as CertV3Lib} from "src/libraries/V3/EigenDACertVerificationV3Lib.sol";
+import {EigenDAV3Cert} from "src/libraries/V3/EigenDATypesV3.sol";
+import {EigenDATypesV2 as DATypesV2} from "src/libraries/V2/EigenDATypesV2.sol";
+import {EigenDATypesV1 as DATypesV1} from "src/libraries/V1/EigenDATypesV1.sol";
+import {EigenDACertVerificationV2Lib as CertV2Lib} from "src/libraries/V2/EigenDACertVerificationV2Lib.sol";
+
+contract EigenDACertVerifierV3 is IEigenDACertVerifier {
+    error InvalidSecurityThresholds();
+
+    IEigenDAThresholdRegistry public immutable eigenDAThresholdRegistry;
+
+    IEigenDASignatureVerifier public immutable eigenDASignatureVerifier;
```

Comment:
curious on your thoughts for why this one isnt made into a library? If you're trying to minimize external contracts needed when deploying, wouldnt this be an easy one?

Reply:
Yes, it is high on my personal priorities. However, I don't think it's the right time since it is a very sensitive, and potentially very complex change.

Reply:
makes sense, will resolve then